### ISSUE TITLE
scrollToFirst props isn't working

### ISSUE THREAD

**Issue by anniewey** [joshjhargreaves/react-native-event-calendar#35]
Is there any workaround to show first event in timeline besides using `scrollToFirst` props?

**Comment by ivanichi** [comment]
yes scrollToFirst = {true} not working on iOS, any solution ? anyone want to discuss here ?
